common/to.etc.alg/src/main/java/to/etc/util/SecurityUtils.java

```diff
@@ -283,29 +283,49 @@ static public boolean checkPassword(@NonNull String encodedPassword, @NonNull St
 		}
 	}
 
-	static public byte[] encryptStringToBytes(String input, String password) throws Exception {
-		SecretKeySpec skeyspec=new SecretKeySpec(password.getBytes(), ENCRYPTION_CIPHER);
-		Cipher cipher=Cipher.getInstance(ENCRYPTION_CIPHER);
+	static private byte[] encryptStringToBytes(String input, String password, Algorythm algorythm) throws Exception {
+		SecretKeySpec skeyspec=new SecretKeySpec(password.getBytes(), algorythm.name());
+		Cipher cipher=Cipher.getInstance(algorythm.name());
 		cipher.init(Cipher.ENCRYPT_MODE, skeyspec);
 		return cipher.doFinal(input.getBytes(StandardCharsets.UTF_8));
 	}
 
-	static public String decryptStringFromBytes(byte[] data, String password) throws Exception {
-		SecretKeySpec skeyspec=new SecretKeySpec(password.getBytes(), ENCRYPTION_CIPHER);
-		Cipher cipher=Cipher.getInstance(ENCRYPTION_CIPHER);
+	static private String decryptStringFromBytes(byte[] data, String password, Algorythm algorythm) throws Exception {
+		SecretKeySpec skeyspec=new SecretKeySpec(password.getBytes(), algorythm.name());
+		Cipher cipher=Cipher.getInstance(algorythm.name());
 		cipher.init(Cipher.DECRYPT_MODE, skeyspec);
 		byte[] decrypted = cipher.doFinal(data);
 		return new String(decrypted, StandardCharsets.UTF_8);
 	}
 
 	static public String encryptString(String input, String password) throws Exception {
-		byte[] bytes = encryptStringToBytes(input, password);
+		byte[] bytes = encryptStringToBytes(input, password, Algorythm.Blowfish);
 		return StringTool.encodeBase64ToString(bytes);
 	}
 
 	static public String decryptString(String input, String password) throws Exception {
 		byte[] bytes = StringTool.decodeBase64(input);
-		return decryptStringFromBytes(bytes, password);
+		return decryptStringFromBytes(bytes, password, Algorythm.Blowfish);
+	}
+
+	static public String encryptStringAes(String input, String password) throws Exception {
+		byte[] bytes = encryptStringToBytes(input, password, Algorythm.AES);
+		return StringTool.encodeBase64ToString(bytes);
+	}
+
+	static public String decryptStringAes(String input, String password) throws Exception {
+		byte[] bytes = StringTool.decodeBase64(input);
+		return decryptStringFromBytes(bytes, password, Algorythm.AES);
+	}
+
+	static public String encryptStringAesHex(String input, String password) throws Exception {
+		byte[] bytes = encryptStringToBytes(input, password, Algorythm.AES);
+		return Hex.encodeHexString(bytes);
```

Comment:
StringTool.toHex()

Reply:
ok, I see, that is why we do not need that library... Ok, I'll switch to that.